sphinx/tutorial/rose/furthertopics/date-time-manipulation.rst

```diff
@@ -96,22 +101,27 @@ The share directory
       <run directory>/share/cycle/<cycle>
 
    These are called the ``share/cycle`` directories.
-
-   The path to the root of the share directory is provided by the
-   ``CYLC_WORKFLOW_SHARE_DIR`` environment variable so the path to the cycle
-   subdirectory would be::
+   ``CYLC_SHARE_CYCLE_DIR`` provides the path to::
 
       "$CYLC_WORKFLOW_SHARE_DIR/cycle/$CYLC_TASK_CYCLE_POINT"
 
-The :ref:`command-rose-task-env` command provides the environment variable
-:envvar:`ROSE_DATAC` which is a more convenient way to obtain the path of the
-``share/cycle`` directory.
 
 To get the path to a previous (or a future) ``share/cycle`` directory we can
-provide an offset to :ref:`command-rose-task-env` e.g::
+provide an offset to isodatetime:
+
+.. code-block:: bash
+
+   isodatetime "${CYLC_TASK_CYCLE_POINT} --offset -P1D
+
+Which can be used to provide custom offset directory locations:
+
+.. code-block:: bash
 
-   rose task-env --cycle-offset=PT1H
+   # Cylc task script
+   CYCLE_POINT_MINUS_P1D=$(isodatetime "${CYLC_TASK_CYCLE_POINT} --offset -P1D)"
+   SHARE_CYCLE_DIR_MINUS_P1D="${CYLC_WORKFLOW_SHARE_DIR}/cycle/${CYCLE_POINT_MINUS_P1D}"
+   mkdir -p "${SHARE_CYCLE_DIR_MINUS_P1D}"
+   echo "I believe in ..." > "${SHARE_CYCLE_DIR_MINUS_P1D}/somefile"
```

Comment:
I don't quite get this example.

Maybe show how to retrieve data from an earlier task rather than creating it.

Reply:
Se if you prefer the new version.